addok_france/utils.py

```diff
@@ -116,14 +119,17 @@ def flag_housenumber(tokens):
 
 def fold_ordinal(s):
     """3bis => 3b."""
-    if s[0].isdigit() and not s.isdigit():
+    if s is not None and s !='' and s[0].isdigit() and not s.isdigit():
```

Comment:
I feel like this needs to be fixed properly beforehand. I'll have a look.

Reply:
No way to reproduce the issue, neither from the shell, the pyshell or the http API.

Can you be a bit more specific on how you get the issue here? A simple way to reproduce from shell or pyshell would help :)

Reply:
This needs either a reproducable test case (so we can understand) either removal :)